### PR TITLE
<Doc> hsm: add document that how to use HSM

### PR DESCRIPTION
**Pull request by LucasLi1024**

### CODE REVIEW

File: 3.x/zh_CN/docs/tutorial/air/use_hsm.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+标签：``硬件加密`` ``HSM`` ``密码机``
+
+---
+
+国密版的FISCO BCOS 3.2.0使用硬件加密模块（Hardware secure module, HSM）进行密码计算的功能，这使得FISCO BCOS拥有更快的密码计算速度，更安全的密钥保护。本教程主主要介绍FISCO BCOS 3.2.0版本的使用方法。
+
+## 1. 节点版本
+
+- 当您的节点需要使用硬件加密模块时，您需要运行硬件加密版fisco-bcos (2.8.0-hsm)节点。通过在节点配置项设置，来启动密码机加密功能。所有密钥对存储在密码机中，内存中无密钥对存留，提高了密钥存储的安全性。
```
Comment (bxq2011hust) [inline 3.x/zh_CN/docs/tutorial/air/use_hsm.md]:
```suggestion
- 当您的节点需要使用硬件加密模块时，需要在节点配置项设置，来启动密码机加密功能。所有密钥对存储在密码机中，内存中无密钥对存留，提高了密钥存储的安全性。
```

File: 3.x/zh_CN/docs/tutorial/air/use_hsm.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+### 第一步. 请根据您密码卡/密码机的安装指引安装好密码机.
+确保将符合了GMT0018-2012规范的头文件和库文件安装在了动态库默认的搜索路径中。比如：
+1. 确保头文件``gmt0018.h``在目录``/usr/include``中，并保证所有用户都有读权限。
+2. 将库文件``libgmt0018.so``放在默认的库搜索路径下，并保证用户具有读和执行权限。可以在节点的配置文件`config.ini`的配置项`security`中配置该so动态库的路径。例如，可以放在Ubuntu操作系统的``/usr/local/lib``目录下，放在CentOS操作系统，``/lib64``或``/usr/lib64``目录下。
+
+### 第二步. 请初始化密码卡/密码机，运行其测试程序确保功能正常.
+请根据密码卡/密码机厂商的指引初始化设备，并创建你所需要的内部密钥。然后运行测试程序，确保功能正常，确保能通过libgmt0018.so的库正确调用密码机所提供GMT0018-2012的接口方法。
+
+## 3. 从源码编译FISCO BCOS 3.2.0
+### 第一步. 获取源码
```
Comment (bxq2011hust) [inline 3.x/zh_CN/docs/tutorial/air/use_hsm.md]: 删除源码编译的环节

File: 3.x/zh_CN/docs/tutorial/air/use_hsm.md (hunk 1 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/tutorial/air/use_hsm.md]: fixed

File: 3.x/zh_CN/docs/tutorial/air/use_hsm.md (hunk 2 above)
Comment (LucasLi1024) [reply 3.x/zh_CN/docs/tutorial/air/use_hsm.md]: has deleted

**Review by bxq2011hust — APPROVED** [review]

**Pull request merged**